web/src/queries.js

```diff
@@ -24,8 +24,8 @@ export const GET_USERS = gql`
 `
 
 export const DELETE_USER = gql`
-mutation deleteUser($id: ID!, $version: Int!){
-  deleteUser(id: $id, version: $version){
+mutation deleteUser($id: ID!){
```

Comment:
I know its a separate thing but I was thinking there is no need for version in create user too, if you want to delete that as part of this change. other than that lgtm.

Reply:
@StephenCoady 👍